encode/vp9enc.c

```diff
@@ -1108,8 +1123,8 @@ vp9enc_write_ivf_header(FILE *vp9_file,
     vp9enc_write_dword(header + 8, VP9_FOURCC);
     vp9enc_write_word(header + 12, width);
     vp9enc_write_word(header + 14, height);
-    vp9enc_write_dword(header + 16, 1);
-    vp9enc_write_dword(header + 20, frame_rate);
+    vp9enc_write_dword(header + 16, frame_rate);
+    vp9enc_write_dword(header + 20, 1);
```

Comment:
As this is a bug fix perhaps it would better to handle it in a different patch, and the timestamp handling in another patch

Reply:
I know that patches should be atomic - but I already had to rewrite the reference management in order to get SVCT working - and for testing purpose it was really handy to have video decodable by the media player ... thats why I included this patch here - if it is a problem I can also export this in a new PR